### PR TITLE
Update CUtlSymbolTable and tier1 project file

### PR DESCRIPTION
**Pull request by zer0k-z**
The tier1 lib itself has not been updated yet.

### CODE REVIEW

File: tier1/tier1.vcxproj.filters
Comment (psychonic) [inline tier1/tier1.vcxproj.filters:1]: Should remove this from the Makefile.tier1 as well. We might as well also remove the cpp file itself as well.

**Comment by GAMMACASE** [comment]
Can you set the default value for the new argument to be ``NULL``?

File: public/tier1/UtlStringMap.h [hunk 1]
```diff
@@ -55,9 +55,9 @@ class CUtlStringMap
 		return m_SymbolTable.Find( pString );
 	}
 
-	UtlSymId_t AddString( const char *pString )
+	UtlSymId_t AddString( const char *pString, bool &result )
```
Comment (GAMMACASE) [inline public/tier1/UtlStringMap.h]: Also why did you used the different name and type in here for the new argument? Is it different to other function prototypes you've got in the pr?

Comment (zer0k-z) [reply public/tier1/UtlStringMap.h]: My bad, the mass replacement didn't quite cover everything.

**Review by GAMMACASE — APPROVED** [review]

**Review by psychonic — APPROVED** [review]
Thanks! I'll update the tier1 libs in the SDK shortly after merge.

**Pull request merged**